psydac/api/ast/utilities.py

```diff
@@ -1042,7 +1042,26 @@ def math_atoms_as_str(expr, lib='math'):
     return set.union(math_functions, math_constants)
 
 def get_name(lhs):
+    """
+    Given a list of variable return the meaninfull part of the name of the
+    first variable that has a _name attribute.
+
+    Was added to solve issue #327 caused by trying to access the name of a 
+    variable that has not such attribute.
+
+    Parameters
+    ----------
+    lhs : list
+        list from whom we need to extract a name.
+
+    Returns
+    -------
+      : str
```

Comment:
If the return quantity does not correspond to a variable, you can just write the type without the column:
```suggestion
    str
```